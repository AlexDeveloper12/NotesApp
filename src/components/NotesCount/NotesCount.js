import React from 'react';
import { Text } from 'react-native-paper';
import { View } from 'react-native';
import PropTypes from 'prop-types';
import commonStyles from '../../styles/CommonStyles/CommonStyles';

function NotesCount({count}) {
    return (
        <View>
            <Text style={[commonStyles.white, commonStyles.robotoLight, commonStyles.commonSize, {marginLeft:12}]}>
                Notes count: {count}
            </Text>
        </View>
    )
}

export default NotesCount;

NotesCount.propTypes = {
    count:PropTypes.number
}

