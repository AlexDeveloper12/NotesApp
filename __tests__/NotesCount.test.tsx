import React from "react"
import NotesCount from "../src/components/NotesCount/NotesCount"
import renderer from 'react-test-renderer';

test('loads and displays note count', async () => {
    const tree = renderer.create(<NotesCount count={5}/>).toJSON();
    expect(tree).toMatchSnapshot();
});
