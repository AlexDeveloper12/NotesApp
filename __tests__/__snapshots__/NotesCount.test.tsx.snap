// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loads and displays note count 1`] = `
<View>
  <Text
    style={
      [
        {
          "textAlign": "left",
        },
        {
          "color": "rgba(28, 27, 31, 1)",
          "fontFamily": "System",
          "fontWeight": "400",
          "letterSpacing": 0,
        },
        {
          "writingDirection": "ltr",
        },
        [
          {
            "color": "white",
          },
          {
            "fontFamily": "Roboto-Light",
          },
          {
            "fontSize": 16,
          },
          {
            "marginLeft": 12,
          },
        ],
      ]
    }
  >
    Notes count: 
    5
  </Text>
</View>
`;
